lib/watir/elements/element.rb

```diff
@@ -468,6 +468,13 @@ def browser
     end
 
     #
+    # Delegates script execution to browser or frame.
+    #
+    def execute_script(script, *args)
```

Comment:
Right now we support `Browser#execute_script` and `IFrame#execute_script` and require the user to pick the one they want to use. 

Unfortunately this gets complicated with this example: `browser.div(id: 'a').iframe(id: 'b').div(id: 'c').select(id: 'd').selected_options` 

Now that we use `#execute_script` to get `#selected_options`, the call needs to be made by the `IFrame`, not the `Browser`, and the resulting elements need to have the frame included in their scoping. The only way I know how to do this is to use the `@query_scope` variable. So in the above example the `Select` instance calls `@query_scope.execute_script`, which means the div element at id: 'c' needs to respond to `#execute_script` by passing it up to its `@query_scope`, and that will happen until the query scope is either an instance of `Browser` or `IFrame`.